justfile

```diff
@@ -224,6 +224,15 @@ compile-ios target:
     cd "$RUST_WORKSPACE"; \
     cargo build --target {{target}} -p xayn-discovery-engine-bindings --release
 
+# Compiles the bindings for the iphoneos (aarch64) and iphonesimulator (x86_64)
+# and copies the binaries to the flutter project
+compile-ios-local:
+    #!/usr/bin/env sh
+    for TARGET in "aarch64-apple-ios" "x86_64-apple-ios"; do
```

Comment:
Nice-to-have: Can we move these to the `.env` file, similar to [here](https://github.com/xaynetwork/xayn_discovery_engine/pull/126/files#diff-e9cbb0224c4a3d23a6019ba557e0cd568c1ad5e1582ff1e335fb7d99b7a1055dR14).

Reply:
done